src/eth_rpc/servers/debug_rpc.rs

```diff
@@ -224,4 +224,24 @@ impl<P: EthereumProvider + Send + Sync + 'static> DebugApiServer for DebugRpc<P>
 
         Ok(tracer.debug_transaction(transaction_hash)?)
     }
+
+    /// Runs an `eth_call` within the context of a given block execution and returns the Geth debug trace.
+    #[tracing::instrument(skip(self), err)]
+    async fn trace_call(
```

Comment:
For now let's keep the request but keep in mind that we might need to remove it in case it logs to much data